### PR TITLE
feat: create InputSelect component

### PR DESCRIPTION
**Pull request by kimak**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/ios/screenshots/InputSelect.png
Comment (kimak) [inline packages/sandbox/e2e/ios/screenshots/InputSelect.png:1]: @ianrobertlambert to review on iOS

File: packages/sandbox/e2e/android/screenshots/InputSelect.png
Comment (kimak) [inline packages/sandbox/e2e/android/screenshots/InputSelect.png:1]: @ianrobertlambert to review on Android

File: packages/sandbox/e2e/ios/screenshots/InputDatePicker.png
Comment (kimak) [inline packages/sandbox/e2e/ios/screenshots/InputDatePicker.png:1]: @ianrobertlambert your feedback about DatePicker color on iOS

**Comment by ianrobertlambert** [comment]
Thanks for sharing @kimak ! ☺️
My feedback:

- [ ] Can we please use a 16 x 16 chevron?
- [ ] Is it possible to transition into the Focus state once the `Default` input is tapped. Currently we just trigger the alert with no indication that the InputSelect is in the `Focus` state.
- [ ] Can we please align the caption across all components in the sandbox? Just for consistency? 😇 I believe we use “Description” elsewhere but here we use “error”.

**Comment by kimak** [comment]
> Thanks for sharing @kimak ! ☺️ My feedback:
> 
> * [x]  Can we please use a 16 x 16 chevron?

good catch! Done ✅

> * [ ]  Is it possible to transition into the Focus state once the `Default` input is tapped. Currently we just trigger the alert with no indication that the InputSelect is in the `Focus` state.

Actually, this state will be handled by the parent component to avoid 2 sources of truth. 
 Then I agree need of focus, it will be done in a later component. 

> * [x]  Can we please align the caption across all components in the sandbox? Just for consistency? 😇 I believe we use “Description” elsewhere but here we use “error”.

🕵️ done ^^

Screenshots are up to date so you can review them. Or use generated QR Code

**Review by hcourthias — APPROVED** [review]

**Comment by kimak** [comment]
> > Thanks for sharing @kimak ! ☺️ My feedback:
> > 
> > * [x]   Can we please use a 16 x 16 chevron?
> 
> good catch! Done ✅
> 
> > * [ ]   Is it possible to transition into the Focus state once the `Default` input is tapped. Currently we just trigger the alert with no indication that the InputSelect is in the `Focus` state.
> 
> Actually, this state will be handled by the parent component to avoid 2 sources of truth. Then I agree need of focus, it will be done in a later component.
> 
> > * [x]   Can we please align the caption across all components in the sandbox? Just for consistency? 😇 I believe we use “Description” elsewhere but here we use “error”.
> 
> 🕵️ done ^^
> 
> Screenshots are up to date so you can review them. Or use generated QR Code

@ianrobertlambert I will merge this MR. let me know if you have any feedback regarding my changes

**Comment by ianrobertlambert** [comment]
Thanks @kimak !! 👌

**Pull request merged**